source/extensions/filters/http/transformation/inja_transformer.cc

```diff
@@ -545,27 +661,50 @@ void InjaTransformer::transform(Http::RequestOrResponseHeaderMap &header_map,
     }
   }
   // get the extractions
-  std::unordered_map<std::string, absl::string_view> extractions;
+  std::unordered_map<std::string, std::string> extractions;
   if (advanced_templates_) {
     extractions.reserve(extractors_.size());
   }
 
   for (const auto &named_extractor : extractors_) {
     const std::string &name = named_extractor.first;
-    if (advanced_templates_) {
-      extractions[name] =
-          named_extractor.second.extract(callbacks, header_map, get_body);
-    } else {
-      absl::string_view name_to_split = name;
-      json *current = &json_body;
+    const ExtractionApi::Mode mode = named_extractor.second.mode();
```

Comment:
Can this be inlined into the switch statement where it is used?

Reply:
done here: https://github.com/solo-io/envoy-gloo/pull/309/commits/5fcb54524240eac9fb9f56b609e1aa0d829513f4#diff-8c1e447dd62e766fbdd2f928d80d5332bdc2defbfd353a41f35db35fcd3b7deaR690